.github/workflows/ci.yml

```diff
@@ -353,7 +353,7 @@ jobs:
   test-reports:
     name: 'Test reports - ${{ matrix.report }}'
     needs: ['project-jobs', 'project-test-jobs']
-    if: ${{ !cancelled() && github.repository == 'woocommerce/woocommerce' && ( github.event_name != 'pull_request' || needs.identify-jobs-to-run.outputs.needs-code-validation == 'true' ) && needs.project-jobs.outputs.report-jobs != '[]' }}
+    if: ${{ !cancelled() && github.repository == 'woocommerce/woocommerce' && ( github.event_name != 'pull_request' || ( needs.identify-jobs-to-run.outputs.needs-code-validation == 'true' && ! github.event.pull_request.head.repo.fork ) ) && needs.project-jobs.outputs.report-jobs != '[]' }}
```

Comment:
We can skip this job for PRs from forks - the reporting will be part of CI-checks for the merge commit.